solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Dialogs/Shared/CalendarSkillDialog.cs

```
@@ -388,230 +390,319 @@ protected async Task DigestCalendarLuisResult(DialogContext dc, Calendar luisRes
+
+                            break;
+                        }
+
+                    case Calendar.Intent.FindCalendarEntry:
+                    case Calendar.Intent.Summary:
                         {
-                            bool isRelativeTime = IsRelativeTime(entity.EndTime[0], result[0].Value, result[0].Timex);
-                            state.EndTime = isRelativeTime ? TimeZoneInfo.ConvertTime(dateTime, TimeZoneInfo.Local, state.GetUserTimeZone()) : dateTime;
+                            break;
                         }
-                    }
-                }
 
-                if (entity.OriginalStartDate != null)
-                {
-                    var culture = dc.Context.Activity.Locale ?? English;
-                    List<DateTimeResolution> results = RecognizeDateTime(entity.OriginalStartDate[0], culture);
-                    if (results != null)
-                    {
-                        var result = results[results.Count - 1];
-                        if (result.Value != null)
+                    case Calendar.Intent.None:
                         {
-                            var dateTime = DateTime.Parse(result.Value);
-                            var dateTimeConvertType = result.Timex;
 
-                            if (dateTime != null)
-                            {
-                                bool isRelativeTime = IsRelativeTime(entity.OriginalStartDate[0], result.Value, result.Timex);
-                                state.OriginalStartDate = isRelativeTime ? TimeZoneInfo.ConvertTime(dateTime, TimeZoneInfo.Local, state.GetUserTimeZone()) : dateTime;
-                            }
+                            break;
+                        }
+
+                    default:
+                        {
+                            break;
                         }
-                    }
                 }
+            }
+            catch
+            {
+                // put log here
+            }
```

probably shouldn't swallow the exception here?